src/core/core.js

```diff
@@ -9,6 +9,8 @@ import QuestionSubmission from './models/questionsubmission';
 import AnalyticsEvent from './analytics/analyticsevent';
 import FilterRegistry from './filters/filterregistry';
 
+import SearchParams from '../../src/ui/dom/searchparams';
```

Comment:
you shouldn't need to leave and include src again, core is already in src
e.g. `../src/ui/dom/searchparams`